telemetry/specs/INT.mdk

```diff
@@ -724,22 +748,32 @@ INT shim header for TCP/UDP:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |      Type     |    Reserved   |     Length    |   DSCP    |R R|
+   | Type  |NPT|R|R|     Length    |  UDP port, IP Proto, or DSCP  |
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-* Type: This field indicates the type of INT Header following the shim header.
+* Type (4b): This field indicates the type of INT Header following the shim header.
 Two Type values are used: one for the hop-by-hop header type and the other for
 the destination header type (See Section [#sec-int-header-types]).
-* Length: This is the total length of INT metadata header, INT stack and the
+* NPT (Next Protocol Type, 2b): This field is meaningful only when the UDP destination
+port number (INT_TBD) is used to indicate the existence of INT. In the other cases,
+this field must be zero. When UDP destination port is INT_TBD, this field may have
+one of the two values:
+** one (1) indicates that UDP payload is followed by the INT stack, and
```

Comment:
Some clarification of where the INT stack and INT shim header is located for different cases would be useful. Previously INT stack and INT shim header was after TCP/UDP header. This revision is suggesting INT stack is after UDP payload. Is that the case for INT natively in TCP also?

Reply:
>> INT stack is after UDP payload
no that's not the message. I intended the original UDP payload is followed by INT stack, not the other way around. Was the writing confusing?

Reply:
Okay my original text was confusing. Fixed them. Thx.